pingdom/src/main/java/com/pulse/dbcp/DBDetailsRetrieve.java

```diff
@@ -0,0 +1,44 @@
+package com.pulse.dbcp;
+
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class DBDetailsRetrieve {
+    private final static Logger log = LoggerFactory.getLogger(UrlDBInsert.class);
+
+    private String url;
+
+    public DBDetailsRetrieve(String url) {
+        this.url = url;
+    }
+    public ArrayList<String> retrieveDetails(){
+        ArrayList<String> arr = new ArrayList<String>();
```

Comment:
why array to return diff type of data? Use a java object to return in case u wanna return multiple fields.